### PR TITLE
Fix an exception in CrowdStrike Falcon

### PR DESCRIPTION
**Pull request by lvoloshyn-sekoia**

### CODE REVIEW

File: CrowdStrikeFalcon/crowdstrike_falcon/event_stream_trigger.py [hunk 1]
```diff
@@ -220,7 +220,8 @@ def run(self) -> None:
                 while not self.f_stop.is_set():
                     if datetime.utcnow() >= next_refresh_at:
                         self.refresh_stream(refresh_url=self.stream_info["refreshActiveSessionURL"])
-                        next_refresh_at = datetime.utcnow() + timedelta(minutes=25)
+                        time_to_refresh = int(self.stream_info["refreshActiveSessionInterval"])
+                        next_refresh_at = datetime.utcnow() + timedelta(seconds=time_to_refresh)
```
Comment (squioc) [inline CrowdStrikeFalcon/crowdstrike_falcon/event_stream_trigger.py:224]: :+1:

I think you should also change the line 197.

File: CrowdStrikeFalcon/crowdstrike_falcon/event_stream_trigger.py [hunk 2]
```diff
@@ -311,12 +313,15 @@ def __init__(self, *args, **kwargs) -> None:
 
         self.auth_token = None
 
-        self.app_id = f"sio-{time.time()}"
+        self.app_id = self.generate_app_id()
```
Comment (squioc) [inline CrowdStrikeFalcon/crowdstrike_falcon/event_stream_trigger.py:316]: Can you move the generation of the app_id into `EventStreamReader` instead? Thanks

**Review by squioc — CHANGES_REQUESTED** [review]
Thanks for the fixes.
Some changes to apply.

File: CrowdStrikeFalcon/crowdstrike_falcon/event_stream_trigger.py (hunk 2 above)
Comment (lvoloshyn-sekoia) [reply CrowdStrikeFalcon/crowdstrike_falcon/event_stream_trigger.py:316]: But then we won't be able to access it directly from the connector

**Review by squioc — APPROVED** [review]

**Pull request merged**